README.md

```diff
@@ -180,6 +180,14 @@ transparently:
 #[cfg_attr(feature = "cargo-clippy", allow(needless_lifetimes))]
 ```
 
+## Updating rustc
+
+Sometimes, rustc moves forward without clippy catching up. Therefore, if you
+updated rustc, clippy wouldn't work and wouldn't compile for a while.
```

Comment:
Perhaps "Therefore updating rustc may leave clippy in a non-functional state until we fix the resulting breakage."?

Reply:
OK